### DISCUSSION TITLE
How to make extra keys transparent like in v0.118.0?

### DISCUSSION THREAD

**Discussion by xronin01** [termux/termux-app#4221] [General]
![Screenshot_20241020-140603_Termux](https://github.com/user-attachments/assets/58ae8918-9066-472b-b939-edf90e464065)
Has this been removed?

**Comment by agnostic-apollo** [comment]
If you mean extra keys have been removed, then no. You might have disabled them, you can enable them by long holding keyboard button in left drawer.

If by transparent you mean if background color has changed, then no to that too, they are still black.

**Comment by xronin01** [comment]
![Screenshot_20241021-203220_Termux](https://github.com/user-attachments/assets/cb0ef929-e4b8-4c4d-877d-5049d7ba0cb6)

**Reply by xronin01** [reply]
In v0.118.0 there was a little opacity

**Reply by agnostic-apollo** [reply]
A hard black was done in b8c3db0b, currently there is no way to change extra keys colors, there are plans to add support for that in future and other theming, but not anytime soon as lot of things need to be considered.
